lib/active_model/serializer/utils.rb

```diff
@@ -23,7 +42,9 @@ def include_args_to_hash(included)
     when Symbol
       { included => {} }
     when Hash
-      included.each_with_object({}) { |(key, value), hash| hash[key] = include_args_to_hash(value) }
```

Comment:
?
there are no rubocop warnings here. Are you saying to have rubocop prefer the do/end usage? and warn against `{}`

I think usage should depend on complexity and length of line. :-\